backend/node_app/utils/searchUtility.js

```diff
@@ -2445,12 +2430,17 @@ class SearchUtility {
 		}
 	}
 
-	async documentSearch(req, body, clientObj, userId) {
+	async documentSearch(req, body, alias, clientObj, userId) {
 		try {
 			const { getIdList, selectedDocuments, expansionDict = {}, forGraphCache = false, searchType } = body;
 			const [parsedQuery, searchTerms] = this.getEsSearchTerms(body);
+			if (alias._source) {
+				searchTerms.push(alias._source.name);
+				body.parsedQuery = parsedQuery.concat(' ', alias._source.name);
+			} else {
+				body.parsedQuery = parsedQuery;
```

Comment:
the parsedQuery should be original and there should be another field called parsedQuery_abbrev  or something 
in the elasticsearch query constructor it needs to check whether its verbatim to use acronyms or not